File: thing.py
# python thing.py
# author: slab
def calc(op, nums):
	match op:
		case '+':
			return sum(nums)
		case '-':
			return nums[0] - sum(nums[1:])
		case '*':
			result = 1
			for num in nums:
				result *= num
			return result
		case '/':
			result = nums[0]
			for num in nums[1:]:
				result /= num
			return result
		case _:
			raise Exception('Invalid operator')

while True:
	words = input().split()
	print(calc(words[0], list(map(int, words[1:]))))

File: thing.clj
; clj -M thing.clj
; author: slab
(loop []
	(let [[op & nums] (clojure.string/split (read-line) #"\s+")
			nums (map #(Integer/parseInt %) nums)
			result (case op
				"+" (reduce + nums)
				"-" (reduce - nums)
				"*" (reduce * nums)
				"/" (reduce / nums)
				(throw (Exception. "Invalid operator")))]
		(println result))
	(recur))
